package.json

```diff
@@ -4,13 +4,18 @@
   "version": "0.0.1",
   "main": "index.js",
   "workspaces": {
-    "packages": ["packages/*"]
+    "packages": [
+      "packages/*"
+    ]
   },
   "scripts": {
     "build": "CI=true yarn workspaces run build",
     "lint": "CI=true yarn workspaces run lint",
     "test": "CI=true yarn workspaces run test",
     "deploy:staging": "yarn workspaces run deploy:staging"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {
+    "@heroicons/react": "^1.0.6"
```

Comment:
how useful do we think this library is in the future? 

we should debate any additional dependency and ensure there is significant value as a whole outweighing the risks of an added dep.

alternatively, if we think we just need this one SVG, we can always just copy it over directly into the repo and later add the dep if we end up using a lot of different icons.

Reply:
removed